src/test/javascript/grouping.controller.test.js

```diff
@@ -1157,6 +1157,13 @@ describe("GroupingController", () => {
             expect(scope.hasDeptAccount).toBeTrue();
         });
 
+        it("should call $scope.displayDynamicModal() when trying to add a dept account to Owners", () => {
+            spyOn(scope, 'displayDynamicModal');
+            scope.addMembers("owners", memberDept);
+
+            expect(scope.displayDynamicModal).toHaveBeenCalled();
+        });
```

Comment:
This test should within the describe block "gs.getMemberAttributeResults callbacks" starting at line 967. So part of the test that gets stuck is when `groupingsService.getMemberAttributeResults` gets called. To resolve this, you will need these two lines of code after calling `scope.addMembers`:
```
const deptResults = { resultCode: "SUCCESS", invalid: [], results: ['testiwt2'] };
httpBackend.expectPOST(BASE_URL + "members", ['testiwt2']).respond(200, deptResults);
httpBackend.flush();
```
Since groupingsService is making a POST request, this mocks the POST request to return what we need,

Reply:
For context, `groupingsService.getMemberAttributeResults` responds with the invalid and valid uhIdentifiers. I realize now the mock is wrong for `results`, because it should be a list of objects that contain name, uid, and uhUuid.